pkg/manager/volumes/pvc_replacer.go

```diff
@@ -73,6 +73,11 @@ func (p *pvcReplacer) getVolReplaceStatusForComponent(tc *v1alpha1.TidbCluster,
 }
 
 func (p *pvcReplacer) UpdateStatus(tc *v1alpha1.TidbCluster) error {
+	if !tc.IsPVCReplaceEnabled() {
```

Comment:
After this PR, if the user wants to enable PVC replace feature for a TidbCluster, both feature-gate in TiDB Operator and TidbCluster should be enabled, right?

If the feature-gate in TiDB Operator is enabled, and then upgrade TiDB Operator to a version with this PR, then this feature is disabled, this may be unexpected for these users.

Reply:
Yes, that's why I have added in release notes. Also I believe this is not released feature and there is no documentation of this feature so customers are highly unlikely to be using this feature.

Reply:
@anish-db WDYT

Reply:
Yes, LGTM to controlling this per cluster, we where thinking of adding something like this too (Since we may want to selectively disable if we want to use resize feature instead of replace).

Slightly related, when using these features we noticed a few edge cases where it would be useful to customize the "spareReplaceReplicas" which is currently default 1, I wanted to send a PR to make that customizable, just FYI will try to send one for it soon as I get a chance. But that will also be a similar parameter to this in CRD.

Reply:
Here's the proposal for the other change about spareReplaceReplicas that I mentioned: https://github.com/pingcap/tidb-operator/pull/5666